gulpfile.js

```diff
@@ -82,9 +82,9 @@ var config = {
   }
 };
 
-// Sass tasks are divided for performance issues regarding dependencies
-// Sass Build task definition, only ran once
-gulp.task('sass:build', ['webfont'], function() {
+//Sass tasks are divided for performance issues regarding dependencies
+// Sass Watch task definition
+gulp.task('sass', function() {
```

Comment:
If you restore this task to his original position (after `sass:dist`), it would be easier to review the changes. (let me know if I cannot explain myself)

Reply:
This review with Andres, and restore the task to original position.